numpy/_core/numeric.py

```diff
@@ -52,8 +52,8 @@
     'isclose', 'isscalar', 'binary_repr', 'base_repr', 'ones',
     'identity', 'allclose', 'putmask',
     'flatnonzero', 'inf', 'nan', 'False_', 'True_', 'bitwise_not',
-    'full', 'full_like', 'matmul', 'shares_memory', 'may_share_memory',
-    '_get_promotion_state', '_set_promotion_state']
+    'full', 'full_like', 'matmul', 'vecdot', 'shares_memory',
+    'may_share_memory', '_get_promotion_state', '_set_promotion_state']
```

Comment:
I wanted `vecdot` to be closer to `matmul` in imports/`__all__` as they are both gufuncs.